src/views/admin-users-listing.phtml

```diff
@@ -61,8 +61,14 @@
                 <thead>
                 <tr>
                     <!--Sorting arrow are displayed using the 'asc','dsc' classes-->
-                    <th scope="col" class="sorting-header asc dsc">ID</th>
-                    <th scope="col">Email</th>
+                    <th>ID</th>
+                    <th scope="col" class="sorting-header <?php
+                        if ($urlHelper->getValue("sort") === "asc"): echo "asc"; endif;
+                        if ($urlHelper->getValue("sort") === "dsc"): echo "dsc"; endif;
```

Comment:
nu-mi plac if-urile astea, dar nu stiu unde sa le pui acum, in url helper nu as zice ca ii locul lor, altfel de helper ti-ar trebui, dar nu il fa in scope-ul pull requestului astuia.